src/users/users.controller.ts

```diff
@@ -283,6 +286,29 @@ export class UsersController {
     }
   }
 
+  @Get('/one/:email/adminpermission')
+  @UseGuards(AuthGuard, AdminGuard)
+  @JwtType(JwtProcessorType.RSA)
+  @ApiOperation({
+    description: SWAGGER_DESC_ADMIN_RIGHTS,
+  })
+  @ApiForbiddenResponse({
+    description: 'user has no admin rights',
+    schema: {
+      type: 'object',
+      properties: {
+        statusCode: { type: 'number' },
+        message: { type: 'string' },
+      },
+    },
+  })
+  @ApiOkResponse({
+    description: 'Returns true if user has admin rights',
+  })
+  async getAdminStatus(@Param('email') email: string): Promise<PermissionDto> {
```

Comment:
`async` is redundant

```suggestion
  getAdminStatus(@Param('email') email: string): Promise<PermissionDto> {
```